src/WrapperApp/components/InputEditor/InputEditorPanel.tsx

```diff
@@ -62,7 +55,7 @@ export default function InputEditorPanel({
 					throw new Error('Unknown generator: ' + generator);
 			}
 		},
-		[controller.signal, convertJSON, convertToInputFiles, generator, simulator]
+		[controller.signal, convertJSON, generator, simulator]
 	);
```

Comment:
Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx